meta-balena-warrior/README.md

```diff
@@ -0,0 +1,11 @@
+# Resin.io layer for Poky Warrior supported boards
```

Comment:
This should be the first commit as that is the one you are working on with this PR. Also, I'd call it ""meta-balena-warrior: Add new layer based on...""

Reply:
reworded the commit message. and reordered them